plugins/identities/http.go

```diff
@@ -30,6 +33,10 @@ type HTTPAuthConfig struct {
 
 	// bearer
 	Token string `mapstructure:"token,omitempty" json:"token,omitempty" yaml:"token,omitempty" validate:"required_if=Type bearer"`
+
+	// oidc
+	Audience                        string `mapstructure:"audience,omitempty" json:"audience,omitempty" yaml:"audience,omitempty" validate:"required_if=Type oidc"`
+	GoogleServiceAccountCredentials string `mapstructure:"google_service_account_credentials,omitempty" json:"google_service_account_credentials,omitempty" yaml:"google_service_account_credentials,omitempty" validate:"required_if=Type oidc"`
```

Comment:
Can you add a comment here for this field to specify if its base64 encoded or jsonifed string